htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -537,56 +481,66 @@
 
 		print '<tr class="oddeven">';
 
-		// date_sync
-		if (!empty($arrayfields['cr.date_sync']['checked'])) {
-			print '<td class="tdoverflowmax200">';
-			print $obj->date_sync;
-			print "</td>\n";
-			if (!$i) {
-				$totalarray['nbfield']++;
+
+		// USER REQUEST UPDATE FOR THIS LINE
+		if ($action == "updateRate" && $obj->rowid == $id_rate_selected) {
+			//  var_dump($obj);
+			$form = new Form($db);
+			print '<td>' . $form->selectDate($obj->date_sync, 'dateinput', 0, 0, 1).'</td>';
+
+
+			print '<td>' . $form->selectMultiCurrency($obj->code, 'multicurrency_code', 1, " code != '".$conf->currency."'", true) . '</td>';
+			print ' <td><input type="number" min ="0" step="any" class="minwidth200" name="rateinput" value="' . dol_escape_htmltag($obj->rate) . '"></td>';
```

Comment:
type="number" : Does this mean the rate is an integer.
This HTML5 type must be avoided because it means the check on number syntax and decimal separator depends on the browser language and does not follow Dolibarr setup (that may be different).